src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -319,21 +319,26 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
 
         const data = depositWithExpiry({
           vault: getAddress(inboundAddressData.address),
-          asset: isToken(fromAssetId(assetId).assetReference)
-            ? getAddress(fromAssetId(assetId).assetReference)
-            : // Native EVM assets use the 0 address as the asset address
-              // https://dev.thorchain.org/concepts/sending-transactions.html#admonition-info-1
-              zeroAddress,
+          asset:
+            // The asset param is a directive to initiate a transfer of said asset from the wallet to the contract
+            // which is *not* what we want for withdraws, see
+            // https://www.tdly.co/shared/simulation/6d23d42a-8dd6-4e3e-88a8-62da779a765d
+            isToken(fromAssetId(assetId).assetReference) && isDeposit
+              ? getAddress(fromAssetId(assetId).assetReference)
+              : // Native EVM asset deposits and token withdraws use the 0 address as the asset address
+                // https://dev.thorchain.org/concepts/sending-transactions.html#admonition-info-1
+                zeroAddress,
           amount: amountOrDustCryptoBaseUnit,
           memo,
           expiry: BigInt(dayjs().add(15, 'minute').unix()),
         })
 
         return {
           // amountCryptoPrecision is always denominated in fee asset - the only value we can send when calling a contract is native asset value
-          amountCryptoPrecision: isToken(fromAssetId(assetId).assetReference)
-            ? '0'
-            : fromBaseUnit(amountOrDustCryptoBaseUnit, feeAsset.precision),
+          amountCryptoPrecision:
+            isToken(fromAssetId(assetId).assetReference) && isDeposit
```

Comment:
actual tx value should still be 0 for tokens regardless of deposit or withdrawal
```suggestion
            isToken(fromAssetId(assetId).assetReference)
```

Reply:
That was the actual issue at hand here, broadcasted many many USDC on AVAX withdraws and couldn't get them to be seen. After comparing against TS, noticed they use the exact same dust value as we have defined in constants, adding it made the Txs finally be seen

Reply:
I see, thought it was related to gas estimation. can we update the comment to give context here then for withdrawal?

Reply:
Yeah the docs definitely don't seem up-to-date here, they're also not mentioning all assets, only native ones so it's really a guessing game (though savers one are/were a bit more furnished) which makes it so we got it right

Reply:
> I see, thought it was related to gas estimation. can we update the comment to give context here then for withdrawal?

0e0778e5d63cbcb1822b1d506d06b8601a060f26